R/mediate.R

```diff
@@ -54,40 +57,56 @@
 #' another variable in colData (argument \code{mediator}) or an assay
 #' (argument \code{assay.type}) or a reducedDim (argument \code{dimred}).
 #' Importantly, those three arguments are mutually exclusive.
-#' 
-#' @return 
+#'
+#' @return
 #' \code{getMediation} returns a \code{data.frame} of adjusted p-values and
 #' effect sizes for the ACMEs and ADEs of every mediator given as input, whereas
 #' \code{addMediation} returns an updated
 #' \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
 #' instance with the same \code{data.frame} stored in the metadata as
-#' "mediation". Its columns include:
-#' 
+#' "mediation" or as specified in the \code{name} argument. Its columns include:
+#'
 #' \describe{
 #'   \item{Mediator}{the mediator variable}
-#'   \item{ACME_estimate}{the Average Causal Mediation Effect (ACME) estimate}
-#'   \item{ADE_estimate}{the Average Direct Effect (ADE) estimate}
-#'   \item{ACME_pval}{the adjusted p-value for the ACME estimate}
-#'   \item{ADE_pval}{the adjusted p-value for the ADE estimate}
+#'   \item{ACME}{the Average Causal Mediation Effect (ACME) estimate}
+#'   \item{ACME_pval}{the original p-value for the ACME estimate}
+#'   \item{ACME_lower}{the lower bound of the CI for the ACME estimate}
+#'   \item{ACME_upper}{the upper bound of the CI for the ACME estimate}
+#'   \item{ADE}{the Average Direct Effect (ADE) estimate}
+#'   \item{ADE_pval}{the original p-value for the ADE estimate}
+#'   \item{ADE_lower}{the lower bound of the CI for the ADE estimate}
+#'   \item{ADE_upper}{the upper bound of the CI for the ADE estimate}
+#'   \item{Total.coef}{the Total Effect estimate}
```

Comment:
All column names should be in same format

I would prefer all lowercase and underscores, but uppercase is also fine I guess